iree/compiler/Conversion/LinalgToSPIRV/test/workgroup_memory_promotion.mlir

```diff
@@ -32,10 +35,15 @@ module attributes {spv.target_env = #spv.target_env<#spv.vce<v1.3, [Shader], [SP
 
 // -----
 
-
-module attributes {spv.target_env = #spv.target_env<#spv.vce<v1.3, [Shader], [SPV_KHR_storage_buffer_storage_class]>, {max_compute_workgroup_invocations = 128 : i32, max_compute_workgroup_size = dense<[128, 128, 64]> : vector<3xi32>}>} {
-  func @conv_no_padding_tile(%arg0: memref<3x4x3x2xf32>, %arg1: memref<?x?x?x3xf32>, %arg2: memref<?x?x?x2xf32>) {
-    linalg.conv(%arg0, %arg1, %arg2) {dilations = [1, 1], strides = [1, 1]} : memref<3x4x3x2xf32>, memref<?x?x?x3xf32>, memref<?x?x?x2xf32>
+module attributes {
+  spv.target_env =
+    #spv.target_env<#spv.vce<v1.3, [Shader], [SPV_KHR_storage_buffer_storage_class]>,
+                    {max_compute_workgroup_invocations = 128 : i32,
+		     max_compute_workgroup_size = dense<[128, 128, 64]> : vector<3xi32>}>} {
+  func @conv_no_padding_tile(%arg0: memref<3x4x3x2xf32>,
+                             %arg1: memref<?x?x?x3xf32>, %arg2: memref<?x?x?x2xf32>) {
+    linalg.conv(%arg0, %arg1, %arg2) {dilations = [1, 1], strides = [1, 1]}
+      : memref<3x4x3x2xf32>, memref<?x?x?x3xf32>, memref<?x?x?x2xf32>
     return
   }
 }
```

Comment:
Not related to this change, but I'm confused that why there are no checks for this test...

Reply:
Good catch! There should be. I think this sneaked in from a previous change and wasnt caught during review. Ill add the CHECKs in another CL